packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -6,28 +6,42 @@ import { withNamespaces, WithNamespaces } from 'react-i18next'
 import { StyleSheet, Text, View } from 'react-native'
 import CurrencyDisplay from 'src/components/CurrencyDisplay'
 import ExchangeRate from 'src/exchange/ExchangeRate'
-import { CURRENCY_ENUM } from 'src/geth/consts'
+import { CURRENCY_ENUM, CURRENCY_ENUM as Tokens } from 'src/geth/consts'
```

Comment:
Is there any reason to use both `CURRENCY_ENUM` and  the`Tokens` alias in the same file?

Reply:
We should decide on consistent `CURRENCY_ENUM` vs `Tokens` usage then rename throughout

Reply:
Nope, just a mistake! Will fix.
I'm for Tokens personally, but didn't want to do a big rename in this PR :)